### PR TITLE
Improved return data for failures

### PR DESCRIPTION
**Pull request by JonTheNiceGuy**
By adding a "status" and "msg" field in the fortios_* call, you can pass more explicit messages back that don't just advise there's an issue in the repo. By adding a default message, you can overide that later with more context.

### CODE REVIEW

File: ansible_templates/code.jinja [hunk 1]
```diff
@@ -49,6 +49,7 @@ def flatten_multilists_attributes(data):
 
 
 def {{path}}_{{name}}(data, fos):
+    resp = {'status': 'failed', 'msg': 'Insufficient data provided to activate module'}
```
Comment (mamunozgonzalez) [inline ansible_templates/code.jinja]: Thanks for your suggestion. I guess you would want to move this line to "def fortios_{{path}}(data, fos):" instead?

Comment (JonTheNiceGuy) [reply ansible_templates/code.jinja]: Oops! Good spot. Resolved in the patches below!

**Comment by mamunozgonzalez** [comment]
Thanks. Looks good.
Can you resolve the conflicts? Once it is done I'll merge it.

**Comment by JonTheNiceGuy** [comment]
Are we looking good now? I didn't think we had any conflicts?

**Comment by mamunozgonzalez** [comment]
hmmm not sure why but github still complains about it: "Cannot be rebased due to conflicts". 
Did you fetch and rebase from the latest commit in master before sending your last commit?

**Comment by mamunozgonzalez** [comment]
Closing as this is handled now in a forked repository (see README). This repo will be archived.

**Pull request closed without merging**